src/main/java/uk/gov/ons/ctp/response/collection/exercise/service/impl/actionrule/ReminderActionRuleCreator.java

```diff
@@ -44,6 +49,11 @@ public void execute(
     final OffsetDateTime offsetDateTime = OffsetDateTime.ofInstant(instant, ZoneId.systemDefault());
     final CollectionExercise collectionExercise = collectionExerciseEvent.getCollectionExercise();
 
+    final ActionPlanDTO activeActionPlan =
+        actionSvcClient.getActionPlanBySelectors(collectionExercise.getId().toString(), true);
+    final ActionPlanDTO inactiveActionPlan =
```

Comment:
could remove .toString in a few places if you pass by UUID instead of String. Being really pick though soz

Reply:
nw cleaner 🎨